wagtail/admin/compare.py

```diff
@@ -104,17 +109,27 @@ def htmldiff(self):
 
 
 class StructBlockComparison(BlockComparison):
+    def htmlvalue(self, val):
+        htmlvalues = {}
```

Comment:
[Until Python 3.7](https://mail.python.org/pipermail/python-dev/2017-December/151283.html), preservation of dict insertion order is not guaranteed, so iterating over child blocks and then doing `htmlvalues.items()` below isn't guaranteed to generate the correct order. Would it be better to use a list of `(name, html)` tuples here instead?

Reply:
A list of tuples is definitely a more appropriate type to use here too, thanks!